jme3-effects/src/main/java/com/jme3/water/WaterFilter.java

```diff
@@ -120,7 +122,9 @@
     private Vector3f center;
     private float radius;
     private AreaShape shapeType = AreaShape.Circular;
-    
+
+    private boolean needSaveReflectionScene;
```

Comment:
@Nehon so, I have added the flag to have a possibility to save a reflection scene if a user knows what he does¸ but in the most cases, we shouldn't do it. I need to have it in some case when the filter is a part of some scene.